torch/csrc/jit/script/compiler.cpp

```diff
@@ -616,11 +616,11 @@ struct to_ir {
     return {def.name().name(), "", std::move(arguments), std::move(returns)};
   }
 
-  std::vector<IValue> evaluateDefaults(
+  c10::impl::GenericList evaluateDefaults(
```

Comment:
why do you need to change places like this? wouldn't vector<Ivalue> here be just fine?

Reply:
it needs an additional conversion but I changed it.